src/pages/TrafficList/TrafficListComponent.tsx

```diff
@@ -0,0 +1,335 @@
+import * as React from 'react';
+import {
+  Badge,
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateVariant,
+  Title,
+  Tooltip,
+  TooltipPosition
+} from '@patternfly/react-core';
+import { style } from 'typestyle';
+import {
+  cellWidth,
+  IRow,
+  ISortBy,
+  sortable,
+  SortByDirection,
+  Table,
+  TableBody,
+  TableHeader
+} from '@patternfly/react-table';
+import { Link } from 'react-router-dom';
+import { TrafficItem, TrafficNode } from './TrafficDetails';
+import * as FilterComponent from '../../components/FilterList/FilterComponent';
+import { ThresholdStatus, NA } from 'types/Health';
+import { NodeType, hasProtocolTraffic, ProtocolTraffic } from 'types/Graph';
+import { getTrafficHealth } from 'types/ErrorRate';
+import history, { URLParam } from 'app/History';
+import { createIcon } from 'components/Health/Helper';
+
+export interface TrafficListItem {
+  healthStatus: ThresholdStatus;
+  icon: string;
+  node: TrafficNode;
+  protocol: string;
+  trafficRate: string;
+  trafficPercent: string; // percent success
+}
+
+type TrafficListComponentProps = FilterComponent.Props<TrafficListItem> & {
+  direction: 'inbound' | 'outbound';
+  trafficItems: TrafficItem[];
+};
+
+type TrafficListComponentState = FilterComponent.State<TrafficListItem> & {
+  sortBy: ISortBy;
+};
+
+const columns = [
```

Comment:
Now with this change of using directly Table instead of the previous component, you can adjust the columns to your needs.
I.e. putting status first with a short width and adding more space to those columns it may need more space, and reducing some fields that you need width would be short.

This kind of things can't be perform in the VirtualList component for a specific component, now it gives you more flexibility.

Reply:
@lucasponce Do you recomment Traffic Status in the first column?  I was OK with the current column ordering but am open to changes.  I will look at the widths and make any adjustments I see fit.